src/script/main/app.ts

```diff
@@ -398,6 +406,15 @@ export class App {
         await e2eiHandler.attemptEnrollment();
       }
 
+      if (dlPathNotifier) {
+        localStorage.setItem(
+          'enforcedDownloadLocation',
+          teamFeatures[FEATURE_KEY.ENFORCE_DOWNLOAD_PATH]?.config.enforcedDownloadLocation ?? '',
+        );
+        const {modalOptions, modalType} = getModalOptions({type: ModalType.DOWNLOAD_PATH_CHANGED});
+        PrimaryModal.show(modalType, modalOptions);
+      }
+
```

Comment:
Let's avoid adding inline code to the app.ts file. Let's extract that to a team config handler (like we did for e2ei) and have the code of that feature living there

Reply:
makes sense